_extension/src/client.ts

```diff
@@ -119,10 +122,32 @@ export class Client {
         await this.client.start();
         vscode.commands.executeCommand("setContext", "typescript.native-preview.serverRunning", true);
         this.onStartedCallbacks.forEach(callback => callback());
-        return new vscode.Disposable(() => {
-            if (this.client) {
-                this.client.stop();
+
+        const codeLensLocationsCommand = vscode.commands.registerCommand("typescript.codeLens.showLocations", (...args: unknown[]) => {

```

Comment:
Is this needed? We could just run `editor.action.showReferences` directly if the editor is VS Code.

Reply:
We cannot because that command concretely expects a `vscode.Uri`, a `vscode.Position`, and `vscode.Location`s. If you try to invoke it with plain LSP objects, it will error.

Reply:
Reallyyyyyyyy 🫠